### DISCUSSION TITLE
Welcome to aws-sdk-qt Discussions!

### DISCUSSION THREAD

**Discussion by pcolby** [pcolby/aws-sdk-qt#23] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by TruePadawan** [comment]
@pcolby I'm trying to build a Qt console application that puts a file on Amazon AWS S3, can you give me an idea on how that would work using this SDK?
